src/main/java/com/googleinterns/zoomtube/utils/LectureUtil.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.googleinterns.zoomtube.data.Lecture;
+
+/** Utility for creating Lecture Entities and Lectures. */
+public final class LectureUtil {
+  /** Used to create a Lecture Entity and its fields. */
```

Comment:
comments should not explain what they are used for but what they are? These are the Entity fields? Maybe re-write it as /** Lecture entity fields. */